lib/index.js

```diff
@@ -53,7 +53,7 @@ function defaults(options) {
   options || (options = {});
   options.apiKey || (options.apiKey = 'YOUR_API_KEY');
   options.host || (options.host = 'cdn.segment.com');
-  options.ajsPath || (options.ajsPath = '/analytics.js/v1/\' + key + \'/analytics.min.js');
+  options.ajsPath || (options.ajsPath = '/analytics.js/v1/\" + key + \"/analytics.min.js');
```

Comment:
Using double quote so that snippet.max and snippet.min would produce the same quotes. Previously, snippet.min would generate code that looks like "/analytics.js/v1' + key + '/analytics.min.js" which makes it a string literal rather than code that concatenate strings.